### ISSUE TITLE
Menoh-sharp error loading U-Net model from onnx file

### ISSUE THREAD

**Issue by blendy2018** [pfnet-research/menoh-sharp#1]
Hi
I'm trying to run Menoh-sharp with the ONNX file converted from the Keras/Tensorflow model.
However, errors occur when loading the model.
This model is U-Net. And the ONNX file was created using keras2onnx.

The U-Net model contains operators that are Transpose, Sigmoid and Pad.
Menoh-sharp does not support these operators(Transpose, Sigmoid and Pad)?
Can U-Net model written by Chainer run?

The sample model Vgg16 could run.
OS: Windows 10
onnx 1.3.0

Kind
